docs/src/componentDocs/AppBar/examples/CustomContentAppBarExample.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Box, IconButton, Toolbar } from '@mui/material';
+import * as colors from '@brightlayer-ui/colors';
+import { AppBar, Spacer, ThreeLiner } from '@brightlayer-ui/react-components';
+import { getBodyFiller } from '../../../shared';
+import { Download, Menu, MoreVert, Search } from '@mui/icons-material';
+import BackgroundImage from '../images/farm.jpg';
+
+export const CustomContentAppBarExample = (): JSX.Element => (
+    <Box sx={{ m: '16px 0', backgroundColor: colors.white[600], p: 4, overflow: 'hidden' }}>
+        <Box sx={{ mb: 2, overflow: 'hidden', height: 400 }}>
+            <AppBar
+                classes={{ collapsed: 'collapsed', expanded: 'expanded' }}
+                sx={{
+                    '&.collapsed': {
+                        '& .BluiThreeLiner-title': {
+                            fontSize: '1.25rem',
+                            fontWeight: 600,
+                        },
+                        '& .BluiThreeLiner-subtitle': {
+                            fontSize: 0,
+                        },
+                        '& .BluiThreeLiner-info': {
+                            fontSize: '1rem',
+                            fontWeight: 400,
+                            mt: '-0.25rem',
+                        },
+                        '.BluiAppBar-background': {
+                            display: 'none',
+                        },
+                    },
+                    '&.expanded': {
+                        '& .BluiThreeLiner-root': {
+                            top: 64,
+                        },
+                    },
+                    width: 450,
+                    mx: 'auto',
+                    zIndex: 'auto',
+                }}
```

Comment:
I am noticing a lot of these `sx` styles are missing from the code snippet. You either show them all, or you hide everything and tell people to "view full code on GitHub" (which I'd be inclined to do). Showing half of the available styles can be misleading to our users.